terraform/scripts/modify_variable.sh

```diff
@@ -79,6 +79,16 @@ echo "Using ssh forwarded ports: $ssh_forwarded_port1 $ssh_forwarded_port2 $ssh_
 echo "Using http forwarded port: $http_forwarded_port."
 
 echo "Modifying ../environment/main.tf file."
+
+[ "$nfs_network_id" == "" ] && \
+        echo "WARNING: Missing \$nfs_network_id. Commenting out vm_with_pf_rules_with_ssh_access_with_nfs_share module in ../environment/main.tf file." && \
+        awk '/module.*vm_with_pf_rules_with_ssh_access_with_nfs_share/,/}/{$0="#"$0}1' ../environment/main.tf  > ../environment/main.tf_new && \
+        mv ../environment/main.tf_new ../environment/main.tf
+[ "$vsc_network_id" == "" ] && \
+        echo "WARNING: Missing \$vsc_network_id. Commenting out vm_with_pf_rules_with_ssh_access_with_vsc_net module in ../environment/main.tf file." && \
+        awk '/module.*vm_with_pf_rules_with_ssh_access_with_vsc_net/,/}/{$0="#"$0}1' ../environment/main.tf  > ../environment/main.tf_new && \
+        mv ../environment/main.tf_new ../environment/main.tf
```

Comment:
I have a suggestion, just to avoid the copy/pasting just move this code into a function and then just a loop for all the current optional networks (`nfs_netwok` and `vsc_network` for now)

Reply:
moved to funtion by 86880cdd1a1e7ebf93b293619fc65dd619ce1a20